server/index.js

```diff
@@ -1,6 +1,6 @@
 const express = require('express');
 const path = require('path');
-const db = require('../database/index.js');
+const db = require('../database/index');
```

Comment:
What I meant was `const db = require('../database');` should work as javascript will search for an index.js

Reply:
This is not too big of a deal. You can resolve this conversation and merge.

Reply:
Considering that is a convention I follow should be ok. Appreciate your diligence.